mobile.go

```diff
@@ -15,7 +15,7 @@ import (
 
 var (
 	httpClient = &http.Client{
-		Transport: proxied.ChainedThenFrontedWith("d2yl1zps97e5mx.cloudfront.net", ""),
+		Transport: proxied.ChainedThenFrontedWith(""),
```

Comment:
While we're in here, it's strange that this depends directly on the proxied package at all. Either we should be passing in a Transport, or the autoupdate package should just move into flashlight.